src/utils/optionsHaveIgnoredMediaFeatureName.js

```diff
@@ -0,0 +1,17 @@
+import matchesStringOrRegExp from "./matchesStringOrRegExp"
+
+/**
+ * Check if an options object contains a user-defined string or regex that
+ * matches the passed in Node.
+ *
+ * @param {object} options
+ * @param {Node} postcss node
+ * @return {boolean} If `true`, a match was found
+ */
+export default function optionsHaveIgnoredAtRule(options, mediaFeatureNode) {
```

Comment:
This should accept the string name directly, not the Node. That way the utility is not dependent on the particular parser.